bouncerscript/test/test_utils.py

```diff
@@ -210,6 +211,60 @@ def test_do_failed_with_TimeoutError_api_call(context, mocker, event_loop, fake_
     assert await api_add_location(context, product, os, path) == expected
 
 
+# api_show_product {{{1
+@pytest.mark.parametrize("product,provided,expected", ((
+    "fake-Fennec-product",
+    "<product>fake-product</product>",
+    "<product>fake-product</product>",
+), (
+    "fake-Firefox-product",
+    "<products/>",
+    "<products/>",
+)))
+@pytest.mark.asyncio
+async def test_api_show_product(context, mocker, product, provided, expected):
+    async def fake_api_call(context, location, data):
+        return provided
+
+    mocker.patch.object(butils, 'api_call', new=fake_api_call)
+    assert await api_show_product(context, product) == expected
+
+
+# api_show_location {{{1
+@pytest.mark.parametrize("product,provided,expected", ((
```

Comment:
Nit: I'd suggest to make `provided` and `expected` the same variable. I got confused on whether the 2 XMLs were intended to be different.